docs/versioned_docs/v0.8/advanced/contexts.md

```diff
@@ -53,6 +53,7 @@ where the context value was provided.
 To access the context, use the `use_context` method.
 
 ```rust
+#[allow(non_snake_case)]
```

Comment:
This is applied by the component macro, and is redundantly used here.

Reply:
Updated.

Reply:
You've made this change other places too where the same suggestion applies. 🙂